### PR TITLE
Some cleanup of UnitOfWork

### PR DESCRIPTION
**Pull request by wimvelzeboer**
I removed duplications of the validation checks, now the methods are more in one level of abstraction.
Also fixed some typos and some minor PDM stuff, mostly spaces and brace positions.

### CODE REVIEW

**Comment by afawcett** [comment]
@agarciaffdc these looks good to me. I added a review as well.

File: fflib/src/classes/fflib_SObjectUnitOfWork.cls [hunk 1]
```diff
@@ -628,14 +590,54 @@ public virtual class fflib_SObjectUnitOfWork
         }
     }
 
+	private void throwExceptionOnEventSObjectType(String sObjectType)
```
Comment (agarciaffdc) [inline fflib/src/classes/fflib_SObjectUnitOfWork.cls]: @wimvelzeboer nice enhancement. Is it possible to use another name on these methods? Something like checkException<blah> It confuses to me a bit. Name seems to say you will always throw an exception but this is not the case if the condition doesn't match.

**Comment by wimvelzeboer** [comment]
@agarciaffdc & @afawcett I have updated this PR and also refactored the commitWork method, to improve readability.

File: fflib/src/classes/fflib_SObjectUnitOfWork.cls [hunk 2]
```diff
@@ -120,13 +124,13 @@ public virtual class fflib_SObjectUnitOfWork
...
 
-        for(Schema.SObjectType sObjectType : m_sObjectTypes)
+        for (Schema.SObjectType sObjectType : m_sObjectTypes)
         {
             // register the type
             handleRegisterType(sObjectType);
         }
 
-        m_relationships.put( Messaging.SingleEmailMessage.class.getName(), new Relationships());
+		m_relationships.put(Messaging.SingleEmailMessage.class.getName(), new Relationships());
```
Comment (dbtavernerffdc) [inline fflib/src/classes/fflib_SObjectUnitOfWork.cls]: In the file view, this line looks too indented.

**Comment by agarciaffdc** [comment]
@wimvelzeboer thanks for your latest contribution. Really appreciate it.

**Pull request merged**